uw-frame-components/portal/widgets/partials/rss-widget.html

```diff
@@ -0,0 +1,29 @@
+<div class="rss">
+  <div ng-if="data.length == 0" layout="row" layout-align="center center">
+    <loading-gif data-object="data" data-empty="isEmpty"></loading-gif>
+  </div>
+  <ul class="widget-list">
+    <li ng-repeat="item in data.items | filter:filterText | limitTo:config.lim">
+      <a class="full-width" ng-href="{{ item.link }}" target="_blank" rel="noopener noreferrer">
+        <div class="headline" ng-class="{ 'nodate' :  !config.showdate }">
+          {{ item.title | truncate:config.titleLim }}
+        </div>
+        <div class="date bold" ng-if="config.showdate">
+          <span>{{ getPrettyDate(item.pubDate) | date:config.dateFormat }}</span>
+        </div>
+      </a>
+    </li>
+    <li ng-if="config.showShowing && config.lim && data.items.length > config.lim" class="no-highlight">
+      <p>Showing {{ config.lim }} of {{ data.items.length }}</p>
+    </li>
+    <li ng-if="error" class="no-highlight">There was an issue loading the information, click see all for more details
+    </li>
+    <li ng-if="isEmpty" class="no-highlight">No information at this time.</li>
+  </ul>
+</div>
+<a target="{{ config.target }}"
```

Comment:
Could this be more secure if it included [`rel="noopener"`](https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/)?